pandas/core/indexes/datetimelike.py

```diff
@@ -82,21 +82,59 @@ class TimelikeOps(object):
 
     _round_doc = (
         """
-        %s the index to the specified freq
+        {op} the index to the specified `freq`.
 
         Parameters
         ----------
         freq : freq string/object
+            The frequency level to {op} the index to.
 
         Returns
         -------
-        index of same type
+        index of same type.
 
         Raises
         ------
-        ValueError if the freq cannot be converted
+        ValueError if the `freq` cannot be converted.
+
+        Notes
+        -----
+        See the following link for possible `freq` values
+        and their descriptions.\n
+        https://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases
+
+        Examples
+        --------
+        >>> rng = pd.date_range('1/1/2018 11:59:00', periods=3, freq='min')
+        >>> rng
+        DatetimeIndex(['2018-01-01 11:59:00', '2018-01-01 12:00:00',
+               '2018-01-01 12:01:00'],
```

Comment:
This should be formatted like
    >>> rng
    DatetimeIndex(['2018-01-01 11:59:00', '2018-01-01 12:00:00',
    ...            '2018-01-01 12:01:00'],
    ...           dtype='datetime64[ns]', freq='T')

so that the doctests pass. (`...` on continuation lines and line up the values). Same with the rest.